<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1855</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1461</width>
      <height>741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-120</y>
        <width>1691</width>
        <height>801</height>
       </rect>
      </property>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>1171</width>
         <height>561</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_1">
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Расход</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_1">
          <item>
           <widget class="QLineEdit" name="lineEdit_1"/>
          </item>
          <item>
           <widget class="QLabel" name="label_1">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>л/cек</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Параметры раствора </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Модель</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Тип</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Плотность </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>ПВ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_20">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>ДНС</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="comboBox_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Бингам</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Гершель-Балкли</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Степенная </string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_4">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Раствор на водной основе</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item>
             <widget class="QLabel" name="label_21">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>г/см^3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
            <item>
             <widget class="QLabel" name="label_22">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>cP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item>
             <widget class="QLabel" name="label_23">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>фунт/100*фут^2*м</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>1090</x>
         <y>720</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ввести данные</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>1281</width>
        <height>891</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>1251</width>
         <height>641</height>
        </rect>
       </property>
       <property name="rowCount">
        <number>200</number>
       </property>
       <property name="columnCount">
        <number>6</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Глубина по стволу(м)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Зенитный угол(град)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Азимут </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Азимут истиный(град)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Азимут магнитный(град)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Глубина по вертикали(м)</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>1100</x>
         <y>657</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Очистить таблицу</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-11</y>
        <width>1851</width>
        <height>771</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>1280</x>
         <y>660</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Добавить строку</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>1100</x>
         <y>660</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Удалить строку</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>1451</width>
         <height>561</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <property name="columnCount">
        <number>11</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <row/>
       <column>
        <property name="text">
         <string>Наименование</string>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignJustify|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Обозначение</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Диаметр внутренний(мм)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Диаметр наружний(мм)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Диаметр максимальный(мм)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тип резьбы низ</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Резьба низ</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тип резьбы вверх</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Резьба вверх</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Длина(м) </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Вес(кг)</string>
        </property>
       </column>
       <item row="0" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>1421</width>
        <height>701</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>1401</width>
         <height>331</height>
        </rect>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <row/>
       <column>
        <property name="text">
         <string>Название</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Индексация</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Вверх(м)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Низ(м)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Коэффициент кавернозности</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>1230</x>
         <y>360</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Добавить строку</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>1030</x>
         <y>360</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Удалить строку</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>730</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Следующая страница</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>720</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Предыдущая страница</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
